packages/yourloops/pages/hcp/index.tsx

```diff
@@ -27,76 +27,49 @@
-import InvalidRoute from "../../components/invalid-route";
 import ProfilePage from "../profile";
 import NotificationsPage from "../notifications";
 import PrimaryNavBar from "./primary-nav-bar";
 import PatientListPage from "./patients/page";
 import TeamsPage from "./teams-page";
 import CertifyAccountPage from "./certify-account-page";
-
-const defaultURL = "/professional/patients";
-const preferencesUrl = "/professional/preferences";
-const log = bows("HcpPage");
+import InvalidRoute from "../../components/invalid-route";
 
 /**
  * Health care professional page
  */
 function HcpPage(): JSX.Element {
-  const { t } = useTranslation("yourloops");
-  const historyHook = useHistory();
   const authHook = useAuth();
 
   const user = authHook.user;
-  const { pathname } = historyHook.location;
 
   if (user === null) {
     throw new Error("User must be looged-in");
   }
 
-  React.useEffect(() => {
-    if (user.role !== UserRoles.hcp) {
-      // Only allow hcp for this route
-      setPageTitle();
-      log.info("Wrong page for current user");
-      historyHook.replace(user.getHomePage());
-    } else if (!/^\/professional\/preferences$/.test(pathname) && user.role === UserRoles.hcp && user.shouldUpdateHcpProfession()) {
-      historyHook.replace(preferencesUrl);
-    } else if (/^\/professional\/?$/.test(pathname)) {
-      // We are on the home page (getHomePage) -> redirect to the correct default route
-      // for this user
-      log.info("Redirecting to the patients list");
-      historyHook.replace(defaultURL);
-    }
-  }, [historyHook, user, t, pathname]);
-
   return (
     <TeamContextProvider>
       <PrimaryNavBar />
       <Switch>
-        <Route path={defaultURL} component={PatientListPage} />
-        <Route path="/professional/teams" component={TeamsPage} />
-        <Route path="/professional/preferences" exact={true} component={() => <ProfilePage defaultURL={defaultURL} />} />
-        <Route path="/professional/notifications" exact={true} component={() => <NotificationsPage defaultURL={defaultURL} />} />
-        <Route path={"/professional/certify"} component={() => <CertifyAccountPage />} />
-        <DataContextProvider context={DefaultDataContext}>
-          <Route path="/professional/patient/:patientId">
-            <PatientDataPage prefixURL="/professional/patient" />
-          </Route>
-        </DataContextProvider>
-        <Route path="/professional" exact={true} />
+        <Route exact path="/teams" component={TeamsPage} />
+        <Route exact path="/preferences" component={ProfilePage} />
+        <Route exact path="/notifications" component={NotificationsPage} />
+        <Route exact path="/certify" component={CertifyAccountPage} />
+        <Route path="/patient/:patientId">
+          <DataContextProvider context={DefaultDataContext}>
+            <PatientDataPage />
+          </DataContextProvider>
+        </Route>
+        <Route path="/patients" component={PatientListPage} />
```

Comment:
I can't figure out why some "Route" doesn't have the "exact" attribute. Is there a specific case ?